.gitignore

```diff
@@ -0,0 +1,4 @@
+.bundle
+_site
+vendor
+Gemfile.lock
```

Comment:
I think the `Gemfile.lock` should be checked in. It makes it easier for new developers to get started with the repository and it prevents issues where a new version of a dependency is released and the the application stops working. There is some more information about this in the [Bundler documentation](https://bundler.io/v1.3/rationale.html):

> After developing your application for a while, check in the application together with the Gemfile and Gemfile.lock snapshot.

> This is important: the Gemfile.lock makes your application a single package of both your own code and the third-party code it ran the last time you know for sure that everything worked. Specifying exact versions of the third-party code you depend on in your Gemfile would not provide the same guarantee, because gems usually declare a range of versions for their dependencies.

Reply:
Thanks @zfketch :)

Reply:
I read that on the other PR but was worried about the implications for GH Pages

Reply:
(and am a noob in Ruby)